lib/src/protocol/models/message.dart

```diff
@@ -75,15 +78,22 @@ abstract class Message {
   String? signature;
 
   static Future<Message> parse(String payload) async {
+    // TODO: how to initialize properly?
+    await Validator.initialize();
```

Comment:
would like a better solution for initializing `Validator`... any ideas?

Reply:
You beat me to it, I was just about to suggest possibly using a singleton with a static constructor. Nice work!